converter/internal/staticconvert/staticconvert.go

```diff
@@ -2,35 +2,35 @@ package staticconvert
 
 import (
 	"bytes"
+	"flag"
 	"fmt"
 
 	"github.com/grafana/agent/converter/diag"
 	"github.com/grafana/agent/converter/internal/common"
 	"github.com/grafana/agent/converter/internal/prometheusconvert"
 	"github.com/grafana/agent/pkg/config"
-	"github.com/grafana/agent/pkg/metrics"
 	"github.com/grafana/agent/pkg/river/token/builder"
 	prom_config "github.com/prometheus/prometheus/config"
+
+	_ "github.com/grafana/agent/pkg/integrations/install" // Install integrations
 )
 
 // Convert implements a Static config converter.
 func Convert(in []byte) ([]byte, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
-	var staticConfig config.Config
```

Comment:
Changes to `Convert` were required for integrations to load into memory correctly.